OracleTuxedo/OracleTSAM/README.md

```diff
@@ -0,0 +1,166 @@
+Oracle TSAM Plus on Docker
+===============
+Sample Docker build files to facilitate installation, configuration, and environment setup for DevOps users. This image has also been certified with [Oracle Container Cloud Service (OCCS)](https://cloud.oracle.com/container). For more information about **Oracle Tuxedo System and Applications Monitor Plus (TSAM Plus)** please see the [Oracle TSAM Plus Online Documentation](http://docs.oracle.com/cd/E72452_01/tsam/docs1222/index.html).
+
+This project offers Dockerfile for building:
+ * Oracle TSAM Plus 12c Release 2 (12.2.2) + Rolling Patch 003
+
+## Dependencies
+This project depends on the [Oracle Server JRE 8 Docker Image](../..//OracleJava). So, before you proceed to build this image, make sure the image `oracle/serverjre:8` has been built locally or is accessible in a remote Docker registry.
+
+**IMPORTANT:** You will have to provide the required installation binaries and put them into the `dockerfiles/<version>` folder.
+
+For this image following installation media / binaries are required:
```

Comment:
Create ".download" files as you see them in the OracleWeblogic folder for example. Please follow the same pattern.